DESCRIPTION

```diff
@@ -41,6 +41,7 @@ Suggests:
     rmarkdown,
     testthat (>= 3.0.0),
     tidyr,
+    units,
```

Comment:
We explicitly recommend using `units` to control the units of `radius` and `buffer` (as well as block size in `spatial_block_cv()`, so added to `Suggests` here. It's in Imports for `sf` so it's already installed anyways.